benchmarks/dist_train_linear.sh

```diff
@@ -5,8 +5,8 @@ set -x
 
 CFG=$1 # use cfgs under "configs/benchmarks/linear_classification/"
 PRETRAIN=$2
-PY_ARGS=${@:3} # --resume_from --deterministic
-GPUS=1 # in the standard setting, GPUS=1
+GPUS=$3 # in MoCo, GPUS=8
```

Comment:
I think we should freeze the `GPUS` here since a different number of GPUs will give different results (otherwise imgs_per_batch in the config file is changed accordingly). We may explain it in the comments, e..g, `GPUS=8 # When changing GPUS, please also change imgs_per_batch in the config file accordingly to ensure the total batch size is 256.` Please also modify `benchmarks/srun_train_linear.sh`, thanks.